### ISSUE TITLE
non-unicode link targets are not supported

### ISSUE THREAD

**Issue by aawsome** [rustic-rs/rustic#117]
I create this as a new issue so it will be still visible after #57 is closed.

This is so far not supported in restic, see https://github.com/restic/restic/issues/3311

So, this can be only fixed after restic fixes it or by breaking restic compatibility.
Until this situation is cleared out, this will get a wontfix label.

**Comment by aawsome** [comment]
Seems like support for this in restic is in progress, see https://github.com/restic/restic/pull/3802
